src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { Fragment } from "react";
+import { Link } from "react-router-dom";
+import "./breadcrumbs.css";
+
+export type BreadcrumbsProps = {
+  nodes: {
+    displayName: string;
+    path: string;
+    id: string;
```

Comment:
Do we need an ID on this? Seems the only use is for the React key, which could probably just be the path?